File: src/Server-API(Java)/FollowMe-Server/src/main/java/com/FollowMeServer/Entity/Roboter.java
package com.FollowMeServer.Entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;


@Entity
public class Roboter {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Integer id;

    public String Adresse;

    public String Name;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getAdresse() {
        return Adresse;
    }

    public void setAdresse(String adresse) {
        Adresse = adresse;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

}

File: src/Anwendung-Follow-Me(Kotlin)/app/src/main/java/com/example/followme/Entity/Roboter.kt
package com.example.followme.Entity


//Roboter-Klasse, die Server gefetcht wird
class Roboter {

    private var id: Int? = null

    private var Adresse: String? = null

    private var Name: String? = null

    fun getId(): Int? {
        return id
    }

    fun setId(id: Int?) {
        this.id = id
    }

    fun getAdresse(): String? {
        return Adresse
    }

    fun setAdresse(adresse: String?) {
        Adresse = adresse
    }

    fun getName(): String? {
        return Name
    }

    fun setName(name: String?) {
        Name = name
    }

}